Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInActivity.java

```diff
@@ -200,6 +200,8 @@ public void onAnimationFinished() {
 
     @Override
     public void onPaymentMethodNonceCreated(final PaymentMethodNonce paymentMethodNonce) {
+        mBraintreeFragment.sendAnalyticsEvent("vaulted-card.select");
```

Comment:
`onPaymentMethodNonceCreated` is called on any `BraintreeFragment` tokenization result, not just vaulted cards. 

Might be worth adding a test that given a non-vaulted tokenization this analytic event is not called

Reply:
Gotcha! We will add logic here so that this "vaulted-card.select" is only called when the nonce is of a Card.

That should resolve this issue, correct?

Reply:
Can you try make a PayPal nonce (non-vaulted) and see if it goes in this method?

Reply:
Its an odd callback since it doesn't seem to get a callback on nonce tokenization without vaulting. I think if that is the case and we can prove that with a test I'm happy with it!

Reply:
Good call! Added tests around this here.

https://github.com/braintree/braintree-android-drop-in/pull/130/commits/2f568923786a07c584f25c105076da318256d716

Reply:
Resolved in [this commit](https://github.com/braintree/braintree-android-drop-in/pull/130/commits/d5d48da09ca22f9b928193056ebd66b38328b50a).